megamek/src/megamek/client/ui/swing/MovementDisplay.java

```diff
@@ -3731,6 +3821,38 @@ private synchronized void updateJoinButton() {
                         for (int element : unitsLaunched) {
                             collarChoices.add(currentDropships.elementAt(
                                     element).getId());
+                            //Prompt the player to load passengers aboard the launching ship(s)
+                            Entity en = clientgui.getClient().getGame().getEntity(currentDropships.elementAt(element).getId());
+                            if (en instanceof SmallCraft) {
+                                SmallCraft craft = (SmallCraft) en;
```

Comment:
This looks very similar to 3703-3732, think we can consolidate them into a function?

Reply:
I was able to break that piece out into a (worker?) function